cpp/modmesh/python/wrapper/modmesh/wrap_SimpleArray.cpp

```diff
@@ -161,15 +163,59 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
             ;
     }
 
-    static void copy_array_using_ellipsis(wrapped_type & self, pybind11::array & arr_in)
+    template <typename D /* for destination type */>
+    struct TypeBroadCast
     {
-        const auto left_shape = self.shape();
+        static void broadcast(wrapped_type & arr_out, pybind11::array & arr_in)
+        {
+            using shape_type = small_vector<size_t>;
```

Comment:
You do not need this alias for it was done in line 45:

```cpp
using shape_type = typename wrapped_type::shape_type;
```

And even if we create an alias, the existing `wrapped_type` should be used.